lib/knackhq/client.rb

```diff
@@ -40,22 +43,38 @@ def records(object)
     end
 
     def records_by_page(object, page_number)
-      hash_request = request.objects(object).records
+      hash_request = request
+                     .objects(object)
+                     .records
                      .get(:params => { :page => page_number,
-                                       :rows_per_page => 500 }).to_h
+                                       :rows_per_page => 500 })
+                     .to_h
       return [] if hash_request.empty?
       transform_hash_keys = hash_request[:records]
       symbolize_hash_keys!(transform_hash_keys)
     end
 
     def records_info(object)
-      hash_request = request.objects(object).records
-                     .get(:params => { :rows_per_page => 500 }).to_h
+      hash_request = request
+                     .objects(object)
+                     .records
+                     .get(:params => { :rows_per_page => 500 })
+                     .to_h
       return [] if hash_request.empty?
       hash_request.delete(:records)
       [hash_request]
     end
 
+    def update_record(object, knackhq_id, json)
```

Comment:
All the other requests return an array. I don't mind returning the `:success` or `:fail`.